examples/google-signin/google-signin-iframe.html

```diff
@@ -43,10 +43,10 @@
     };
 
     window.onload = () => {
-        const creator = new SwgGoogleSigninCreator(
+        const iframe = new SwgGsiIframe(
```

Comment:
I prefer the original name because the class doesn't create an iframe, it only will create and handle the buttons on the iframe. 

Also I prefer to spell out Google Sign In instead of GSI since it isn't commonly used.

Reply:
Good call, I named it `SwgGoogleSignInButtonCreator`